src/components/list-option/list-option.jsx

```diff
@@ -31,7 +41,7 @@ const ListOption = forwardRef(function ListOption(props, ref) {
   });
 
   return (<li
-    aria-selected={props.selected}
+    aria-selected={selected}
```

Comment:
Should this be
```suggestion
    aria-selected={active && selected}
```

To ensure that only `active` components will be `aria-selected`?

Reply:
I think active components can be unselected, and selected components can be inactive

Reply:
Perhaps I'm conflating this incorrectly with `focused`.